### DISCUSSION TITLE
Vertical timeline (I AM ON MY KNEES I NEED THIS FOR MY DND WORLD PLZ)

### DISCUSSION THREAD

**Discussion by chiafish** [mermaid-js/mermaid#5549] [Ideas]
Smt like this 
![image](https://github.com/mermaid-js/mermaid/assets/126991555/26083336-550f-4334-977f-e51da3a65152)

**Comment by pajaro1966** [comment]
It's necessary, yes. Surprisingly (or perhaps not), Gantt diagrams surpasses TimeLine in representing timelines. TimeLine diagrams are too limited.

**Reply by pajaro1966** [reply]
The method in some Obsidian plugins for timelines resemble gannt diagrams  and are more useful than mermaid's. Just take a look:
https://community.obsidian.md/plugins/chronos

As of today, mermaid's implementation of timelines is not satisfactory. That said, I do really appreciate the hardwork by its team. Thanks to the team
